boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -118,13 +118,13 @@
 &spi3 {
 	status = "okay";
 
-	cs-gpios = <&gpiod 13 0>, <&gpioe 0 0>;
+	cs-gpios = <&gpiod 13 GPIO_ACTIVE_HIGH>, <&gpioe 0 GPIO_ACTIVE_HIGH>;
 
 	spbtle-rf@0 {
 		compatible = "zephyr,bt-hci-spi";
 		reg = <0>;
-		reset-gpios = <&gpioa 8 0>;
-		irq-gpios = <&gpioe 6 0>;
+		reset-gpios = <&gpioa 8 GPIO_ACTIVE_HIGH>;
+		irq-gpios = <&gpioe 6 GPIO_ACTIVE_HIGH>;
```

Comment:
These changes are not related to eswifi driver. It may be better to do them in another commit.

Reply:
indeed, fixed thanks